docs/src/solvers/index.md

```diff
@@ -1,43 +1,114 @@
 
-# List of available solvers
+# Available solvers in Manopt.jl
 
 ```@meta
 CurrentModule = Manopt
 ```
 
-Solvers can be applied to [`AbstractManoptProblem`](@ref)s with solver
-specific [`AbstractManoptSolverState`](@ref).
+Optimisation problems can be classified with respect to several criteria.
+In the following we provide a grouping of the algorithms with respect to the “information”
+available about your optimisation problem
 
-# List of algorithms
+```math
+\operatorname*{arg\,min}_{p∈\mathbb M} f(p)
+```
+
+Within the groups we provide short notes on advantages of the individual solvers,
+for example whether ``f`` should be convex, or nonconvex – or whether the method works
+locally or even globally.
+
+## Derivative Free
+
+For derivative free only function evaluations of ``f`` are used.
+
+* [Nelder-Mead](NelderMead.md) a simplex based variant, that is using ``d+1`` points, where ``d`` is the dimension of the manifold.
+* [Particle Swarm](particle_swarm.md) use the evolution of a set of points, called swarm, to explore the domain of the cost and find a minimizer.
```

Comment:
```suggestion
* [Particle Swarm](particle_swarm.md) use the evolution of a set of points, called swarm, to explore the domain of the cost and find a minimizer.
* [CMA-ES](cma_es.md) uses a stochastic evolutionary strategy to perform minimization robust to local minima of the objective.
```

Reply:
oh, did I forget that one? Sorry for that.